src/interfaces.ts

```diff
@@ -57,6 +57,8 @@ export interface GenerateRequest {
   images?: Uint8Array[] | string[]
   keep_alive?: string | number // a number (seconds) or a string with a duration unit suffix ("300ms", "1.5h", "2h45m", etc)
   think?: boolean | 'high' | 'medium' | 'low'
+  logprobs?: boolean
```

Comment:
We should add these fields in the README also

Reply:
good callout! will add